addons/website_slides/static/src/js/slides_course_page.js

```diff
@@ -26,7 +26,7 @@ export const SlideCoursePage = publicWidget.Widget.extend({
     collapseNextCategory: function (nextCategoryId) {
         const $categorySection = document.getElementById(`category-collapse-${nextCategoryId}`);
         const $categorySlides = document.querySelector(`ul[id=collapse-${nextCategoryId}]`);
-        if ($categorySection.getAttribute('aria-expanded') === 'false' && !$categorySlides.classList.contains('show')) {
+        if ($categorySection?.getAttribute('aria-expanded') === 'false' && !$categorySlides?.classList.contains('show')) {
```

Comment:
`!undefined` is true, right?

If there is always a `categorySlides` if there is a `categorySection`, we could perhaps simplify with an early return if there is no `categorySection` instead?

 Also maybe we can drop the `$` as these are not jQuery objects?

Another way of dealing with this would be to not call the method at all if not in the fullscreen view, but I suppose that is not an easier option.

Reply:
Yes you're right `!undefined` is true, but the `?` is checking if `$categorySection` is `null`or `undefined` and if it is then the expression will short-circuit and evaluate to `undefined` so this works :slightly_smiling_face: ( `undefined === 'false'` is `false`)
I think I prefer using the `?` here as it is more concise than adding 3 lines for an early return :slightly_smiling_face: 

You're right I can only check for the `categorySection` and drop the `$`. :smile: 

We also need to collapse the next category in this view so it is more than just the fullscreen view : 
![image](https://github.com/odoo/odoo/assets/107679666/317145a7-ad9e-4a86-b319-27a8fd861ae6)

We also get the `nextCategoryId` if the section is completed in every case so there's not really an easier solution than checking if we have the elements to collapse. :sweat_smile: